filebeat/harvester/reader/multiline.go

```diff
@@ -150,9 +146,9 @@ func (mlr *Multiline) readNext() (Message, error) {
 
 				// return collected multiline event and
 				// empty buffer for new multiline event
-				message := mlr.finalize()
+				m := mlr.finalize()
```

Comment:
maybe use `msg := mlr.finalize()` here. Makes it more clear `m` is of type `message`.